CustomBatchWatchdog.cs

```diff
@@ -21,8 +21,8 @@ public partial class CustomBatchWatchdog : ServiceBase
         List<RecoveryItem> recoveryItems = new List<RecoveryItem>();
 
         // local constants (not configurable)
-        const string configFileName = "cbwatchdog.json";
-        const string eventLogSource = "Custom Batch Watchdog";
+        string configFileName = "cbwatchdog.json";
+        string eventLogSource = "Custom Batch Watchdog";
```

Comment:
please specify the default visibility (`private`) to all the fields above.

https://github.com/Starcounter/CompanyTrack/blob/15235f13eee21b97625fe6845063d92d68d63516/AppsTeam/Guidelines/coding_style/csharp_code_formatting.md -> 7

Reply:
Done